pkg/packer/manifests/aws/packer.json.tmpl

```diff
@@ -73,6 +73,7 @@
       "iam_instance_profile": "{{user `iam_instance_profile`}}",
       "skip_profile_validation": "{{user `skip_profile_validation`}}",
       "ssh_username": "{{user `ssh_username`}}",
+      "ssh_key_exchange_algorithms": ["curve25519-sha256@libssh.org", "ecdh-sha2-nistp256", "ecdh-sha2-nistp384", "ecdh-sha2-nistp521", "diffie-hellman-group14-sha1","diffie-hellman-group1-sha1"],
```

Comment:
I think we need to replicate this in:

https://github.com/mesosphere/konvoy-image-builder/blob/main/pkg/packer/manifests/vsphere/packer.json.tmpl
https://github.com/mesosphere/konvoy-image-builder/blob/main/pkg/packer/manifests/vsphere/base/packer-base.json

To make sure that `diffie-hellman-group1-sha1` is an available algo there too